armi/materials/tests/test_materials.py

```diff
@@ -601,15 +622,15 @@ def test_heatCapacity(self):
     def test_getTemperatureAtDensity(self):
         expectedTemperature = 100.0
         tAtTargetDensity = self.mat.getTemperatureAtDensity(
-            self.mat.density(Tc=expectedTemperature), 30.0
+            self.mat.pseudoDensity(Tc=expectedTemperature), 30.0
         )
         self.assertAlmostEqual(expectedTemperature, tAtTargetDensity)
 
     def test_getDensityExpansion3D(self):
         expectedTemperature = 100.0
 
         ref_density = 10.86792660463439e3
-        test_density = self.mat.density3KgM3(Tc=expectedTemperature)
+        test_density = self.mat.pseudoDensityKgM3(Tc=expectedTemperature)
```

Comment:
Looks like this should be `densityKgM3`, yes?

Reply:
done